restore.cmd

```diff
@@ -14,5 +14,3 @@ echo Restoring debug all...
 START /W "Restoring debug all..." CMD /c buildVS2017.cmd d r 1
 echo Restoring Release all... 
 START /W "Restoring release all..." CMD /c buildVS2017.cmd r r 1
-
-echo Restoring WinRT packages should be done from the VS2015 project
```

Comment:
Should this be removed?

Reply:
Minor cleanup, but we no longer need VS 2015 for anything.

Reply:
I think it was a tools issue. I saw it as added....

Reply:
In my diff, it shows up as added again... that's why I ask.

---
In reply to: [208547427](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1175#discussion_r208547427) [](ancestors = 208547427)